packages/lesmis/src/components/DashboardExportModal/components/Header.js

```diff
@@ -14,26 +15,33 @@ const FlexColumn = styled(BaseFlexColumn)`
   text-align: center;
 `;
 
-const Heading = styled(Typography)`
+const BaseHeading = styled.p`
   font-weight: 600;
   line-height: 140%;
   text-transform: capitalize;
+  margin: 0;
+`;
+
+const Heading = styled(BaseHeading)`
+  font-size: 23px;
   color: ${props => props.theme.palette.primary.main};
 `;
 
-const SubHeading = styled(Typography)`
-  font-weight: 600;
-  line-height: 140%;
-  text-transform: capitalize;
+const SubHeading = styled(BaseHeading)`
+  font-size: 17px;
 `;
 
 const Header = ({ dashboardLabel, useYearSelector, selectedYear }) => {
+  const { entityCode } = useUrlParams();
+  const { data: entityData } = useEntityData(entityCode);
+  const entityName = entityData?.name ? entityData?.name : '';
```

Comment:
```suggestion
  const entityName = entityData?.name || '';
```
Slightly simpler, but I don't really mind 👍